draft-ietf-idr-link-bandwidth.xml

```diff
@@ -170,26 +196,34 @@
         egress processing to adj-RIB-out only or in ingress processing in
         which case link bandwidth community is present in Local-RIB.</t>
 
-        <t> Note: Implementation may provide configuration option (knob) to
-        allow sending of non-transitive link bandwidth communities to external
-        BGP sessions.</t>
+        <t>Note: Implementation may provide configuration option (knob) to
+        allow sending of non-transitive link bandwidth extended community
```

Comment:
"to allow sending non-transitive link bandwidth extended community on external BGP sessions."

and MAY caps?

Reply:
I have made this change. Please review.